tests/console/zypper_lr_validate.pm

```diff
@@ -240,11 +240,13 @@ sub validate_repos_sle {
             # there will be no nvidia repo when WE add-on was removed with MIGRATION_REMOVE_ADDONS
             my $addon_removed = uc get_var('MIGRATION_REMOVE_ADDONS', 'none');
             $we = 1 if ($scc_product eq 'SLE-WE' && $scc_product !~ /$addon_removed/);
-            for my $product_channel ("Pool", "Updates", "Debuginfo-Pool", "Debuginfo-Updates", "Source-Pool") {
+            for my $product_channel ("Pool", "Updates", "Debuginfo-Pool", "Debuginfo-Updates", "Source-Pool", "Debug", "Source", "") {
                 # Toolchain module doesn't have Source-Pool channel
+                next if ((is_sle('>=16')) && ($product_channel eq 'Pool' || $product_channel =~ /Updates/));
```

Comment:
Why are you puttingthis between TCM code and comment ?

Reply:
I missed this comment :(  Fixed that and add comment for sle 16.